qa/workunits/cephadm/test_adoption.sh

```diff
@@ -12,6 +12,10 @@ trap "$SUDO rm -rf $TMPDIR" EXIT
 
 [ -z "$SUDO" ] && SUDO=sudo
 
+# combine into a single var
+CEPHADM_BIN="$CEPHADM"
+CEPHADM="$SUDO $CEPHADM_BIN"
```

Comment:
this change practically comments out following code block which set `CEPHADM` to `${CEPHADM_SRC_DIR}/cephadm` if it is empty.

shall we move it after we assign a fallback value to `CEPHADM`?

also, could just put
```sh
CEPHADM="$SUDO $CEPHADM"
```

and why do we want to use sudo here?

Reply:
sigh, fixed.

we need sudo for pretty all cephadm commands.